build_exe/linux/entrypoint-linux.sh

```diff
@@ -8,7 +8,7 @@ set -e
 
 cd /src
 
-pip install .
+pip install commcare-export
```

Comment:
There was an issue where the version is not set, so doing 
> pip install .

don't work. Maybe something to look into in the future; not sure how important it is right now since we're able to simply run `pip install commcare-export` (of course, running `pip install commcare-export` assumes the newest version is already pushed to pypi).

Reply:
I think you need `-e`. And depending on the version of Python, you might need setuptools too:
```
pip install setuptools
pip install -e .
```
Disclaimer: I have not tested this.

Reply:
Thanks for the suggestion. I tried it, but alas...it didn't work.

Reply:
I tried various other ways which was also a dead end or just unfeasible in the long run...I don't _think_ this issue is really blocking, just annoying (let me know if you disagree), so I'll create a ticket to address it.